.env

```diff
@@ -0,0 +1 @@
+API_URL="patv9V491TkNGUhyA.0e452983e9bc1b7541a5bcc3029deb4e6882327d620244e2c44b4399f9700c28",
```

Comment:
Hey @sarthak-stranger Please remove the API key. Remember use `.env` to load your API Key's. Ref. [.env for api](https://stackoverflow.com/questions/49579028/adding-an-env-file-to-a-react-project) and always add `.env` to `.gitignore` so that you didn't accidentally push it into the PR.

Reply:
Ok sir